integration/simulation/validate_chain.go

```diff
@@ -248,6 +249,10 @@ func checkBlockchainOfObscuroNode(t *testing.T, nodeClient *obscuroclient.Client
 	for _, wallet := range s.TxInjector.wallets {
 		total += balance(nodeClient, wallet.Address)
 	}
+
+	// also check for the worker wallet address
```

Comment:
this "worker wallet", what exactly is it? why is it needed?

Reply:
This is to avoid contract deployment and tx injection to be using the same wallet as the nodes.